nix/master.nix

```
@@ -117,6 +117,7 @@ in
         c["www"]["plugins"].update(
             dict(base_react={}, waterfall_view={}, console_view={}, grid_view={})
         )
+        c['services'].append(reporters.Prometheus(port=8011))
```

The port might collide with something else:

```suggestion
        ${lib.optionalString (cfg.prometheusExporterPort != null) ''
          c['services'].append(reporters.Prometheus(port=${cfg.prometheusExporterPort}))
        ''}
```